frontend/packages/erd-core/src/components/ERDRenderer/AppBar/CopyLinkButton/CopyLinkButton.tsx

```diff
@@ -31,12 +31,13 @@ export const CopyLinkButton: FC = () => {
         })
       })
 
-    clickLogEvent({
-      element: 'copyLinkButton',
-      cliVer: cliVersion.version,
-      appEnv: cliVersion.envName,
-    })
-  }, [toast, cliVersion.version, cliVersion.envName])
+    cliVersion.displayedOn === 'cli' &&
```

Comment:
Eventually we would like to measure this in the web version, but that can be done later!